src/styles.css

```diff
@@ -0,0 +1,146 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+html,
+body {
+  height: 100%;
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  margin: 0;
+  scroll-behavior: smooth;
+  min-height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.todo-container {
+  box-shadow: 0 0 1rem rgba(22, 28, 45, 0.2);
+  border-radius: 0;
+}
+
+.card-header {
+  padding-top: 16px;
+}
+
+.card-toolbar svg {
+  width: 20px;
+  color: grey;
+}
+
+form input {
+  width: 95%;
+  padding: 12px 13px;
+  border: none;
+  font-style: italic;
+  font-size: 15px;
+}
+
+form button {
+  border: none;
+  background: transparent;
+  padding: 0 14px;
+}
+
+form button img {
+  width: 15px;
+  transform: scaleX(-1);
+  filter: opacity(0.4);
+}
+
+form input::placeholder {
+  font-size: 15px;
+  font-style: italic;
+  font-weight: 300;
+}
+
+form input:active,
+form input:focus {
+  border: none;
+  outline: none;
+}
+
+.card-footer {
+  padding: 16px;
+}
+
+.card-footer a {
+  text-decoration: none;
+  color: grey;
+  font-weight: 300;
+  font-size: 14px;
+}
+
+.card-footer a:hover {
+  color: #000;
+  text-decoration: underline;
+}
+
+.todo-item {
+  width: 100%;
+}
+
+.list-group-item {
+  padding: 15px 11px;
+}
+
+.todo-item .todo-content {
+  font-size: 15px;
+  font-weight: 200;
+  height: 24px;
+  border: none;
+}
+
+.list-group-item .delete_action_btn {
+  display: none;
+}
+
+.list-group-item.active {
+  background-color: rgb(255 255 0 / 74%) !important;
+  border: none;
+}
+
+.list-group-item.active .todo-content {
+  background-color: rgb(255 255 0 / 74%) !important;
+}
```

Comment:
- [ ] [OPTIONAL] It's a good thing you didn't overuse `!important` 😄 ! As best practice, it is not healthy to use it more than three times because it has the potential of introducing bugs in the code base as well as causing some conflicts in terms of specificity. To avoid such delays, I think it is best to just mind its usage and use `class` selectors mostly for the styles 🤗